src/index.html

```diff
@@ -11,7 +11,160 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+     action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="form"
+      >
+
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="naming">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          >
+        </label>
+        <label class="naming">
+          Name:
+          <input
+          type="text"
+          name="Name"
+          >
+        </label>
+        <label class="age">
+          How old are You?
+          <input
+          type="number"
+          name="age"
+          value="12"
+          class="ageSize"
+          >
```

Comment:
make sure to fix such issues over your code
```suggestion
          <input
            type="number"
            name="age"
            value="12"
            class="ageSize"
          >
```